frontend/src/components/AdminModals/AdminSuspendModal.js

```diff
@@ -27,22 +27,28 @@ function AdminSuspendModal(props) {
     </button>
   );
 
+  const modalText = (isSuspended) => {
+    const textForSuspendedUser = 'Clicking "UnSuspend" will allow the user to logging into the app.'
```

Comment:
It looks like this text is missing spaces between the sentences (after the `.`s).

Also, contrary to @vhoof 's earlier feedback, I think I'd actually prefer to have this be one long string than to use string concatenation (the linter width limit is smart enough to ignore long strings). This is in-keeping with the [airbnb javascript style guide](https://github.com/airbnb/javascript#strings--line-length), and it makes catching little mistakes like this quite a bit easier.

Reply:
Totally fair. Listen to him, not to me. :)